client/src/components/AboutCard/AboutCardsContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Grid} from '@trussworks/react-uswds';
+
+type AboutCardsContainerProps = {
+  className?: string
+}
+
+// note: this assumes a J40MainGridContainer container!
+const AboutCardsContainer = (props: React.PropsWithChildren<AboutCardsContainerProps>) => {
+  return (
+    <Grid row gap={'lg'} className={'j40-aboutcard-container ' + (props.className || '')}>
```

Comment:
In `(props.className || '')`, why is the `or empty` string needed? Seems that since the prop is optional, we may not need that? Not sure tho?

Reply:
@NatHillardUSDS Can chime in, but I think without this `|| ''` you'll get `undefined`

Reply:
if undefined is a type we can add it to the props definition as `string || undefined`

Reply:
I think the html output literally has `undefined` in it. I think in most use cases `props.className` is typed as optional... (e.g. `className?: string`) it can be undefined.